apps/src/lib/client/tx.rs

```diff
@@ -57,8 +50,14 @@ const TX_UNBOND_WASM: &str = "tx_unbond.wasm";
 const TX_WITHDRAW_WASM: &str = "tx_withdraw.wasm";
 const VP_NFT: &str = "vp_nft.wasm";
 
-const ENV_VAR_ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT: &str =
-    "ANOMA_TENDERMINT_WEBSOCKET_TIMEOUT";
+/// Timeout for requests to the `/accepted` and `/applied`
+/// ABCI query endpoints.
+const ENV_VAR_NAMADA_EVENTS_MAX_WAIT_TIME_SECONDS: &str =
+    "NAMADA_EVENTS_MAX_WAIT_TIME_SECONDS";
```

Comment:
for consistency with other env vars, I suggest we make this
```suggestion
    "ANOMA_EVENTS_MAX_WAIT_TIME_SECONDS";
```
(we really should have some central mod for env vars)